Sources/PasscodeKit/Views/KeypadView.swift

```diff
@@ -30,14 +35,42 @@ struct KeypadView: View {
                 NumberButton(value: .text("9"), text: $text)
             }
             HStack(spacing: configuration.hSpacing) {
-                NumberButton(value: .blank, text: $text)
-                    .hidden()
-                    .disabled(true)
+                if passcodeEnv.manager.passcode?.isBiometricsEnabled ?? false {
+                    Button {
+                        onBiometry()
+                    } label: {
+                        ZStack {
+                            Image(systemName: biometricsImage)
+                                .resizable()
```

Comment:
Better to just add `.font(.title)` here, then the image will be the same scale as the rest of the numbers and also remove `.frame(maxWidth: 40, maxHeight: 40)` as this will hardcode the size and therefore disable accessibility scaling.

Reply:
@divadretlaw This is awesome to use dynamic font here!